portfolio/src/main/webapp/tutorial.js

```diff
@@ -3,10 +3,10 @@
 let tutorial = document.getElementsByName('.modal');
 
 function open_tutorial() {
-    tutorial.style.display = 'block';
-    console.log("NO");
+  tutorial.style.display = 'block';
+  console.log('NO');
```

Comment:
https://giphy.com/gifs/the-office-no-steve-carell-12XMGIWtrHBl5e

Reply:
Nooooooooooo